main.go

```diff
@@ -58,12 +59,20 @@ func main() {
 	}
 
 	var cmds []*exec.Cmd
+	// check for a .git directory and whether it's empty
+  if isDirEmpty(filepath.Join(c.Dir, ".git")) {
+		cmds = append(cmds, initGit())
+		cmds = append(cmds, remote(c))
+	}
+
+	cmds = append(cmds, fetch(c))
+
 	if isPR(c) {
-		cmds = append(cmds, clone(c))
-		cmds = append(cmds, fetch(c))
+		//cmds = append(cmds, clone(c))
+		//cmds = append(cmds, fetch(c))
 		cmds = append(cmds, checkoutHead(c))
 	} else {
-		cmds = append(cmds, cloneBranch(c))
+		//cmds = append(cmds, cloneBranch(c))
 		cmds = append(cmds, checkoutSha(c))
```

Comment:
I wonder if we need to have special commands to clone a tag? Technically a tag is just a sha, so it should work ... just something I'd like to verify before merging. If you can confirm I'm good to merge. I'm happy to test as well, but I'm offline until Monday :(